src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1103,31 +1103,48 @@
     (dissoc row-map :environment_id)
     row-map))
 
+(defn normalize-resource-event
+  "Prep `event` for comparison/computation of a hash"
+  [event]
+  (-> event
+      (utils/update-when [:timestamp] to-timestamp)
+      (utils/update-when [:old_value] sutils/db-serialize)
+      (utils/update-when [:new_value] sutils/db-serialize)
+      (assoc :containing_class (find-containing-class (get event :containment_path)))))
```

Comment:
How come you use `(get event :containment_path)` instead of `(:containment_path event)`?